buildIncremental.fsx

```diff
@@ -42,18 +44,37 @@ module IncrementalTests =
         | EndsWith "cmd" -> true
         | EndsWith "sh" -> true
         | _ -> false
+    
+    let getHeadHashFor repositoryDir branch =
+        let _, msg, error = runGitCommand repositoryDir (sprintf "log --oneline -1 %s" branch)
+        if error <> "" then failwithf "git log --oneline failed: %s" error
+        let logMsg = msg |> Seq.head
+        let tmp =
+            logMsg.Split(' ')
+            |> Seq.head
+            |> fun s -> s.Split('m')
+        if tmp |> Array.length > 2 then tmp.[1].Substring(0,6) else tmp.[0].Substring(0,6)
 
     let getUpdatedFiles() = 
         let srcDir = __SOURCE_DIRECTORY__
         let localBranches = getLocalBranches srcDir
-        if not (localBranches |> Seq.exists (fun b -> b = "v1.3")) then
-            directRunGitCommandAndFail srcDir "fetch origin v1.3:v1.3"
-        let forkPoint = runSimpleGitCommand srcDir "merge-base --fork-point v1.3"
+        log "Local branches..."
+        localBranches |> Seq.iter log
+        if not (localBranches |> Seq.exists (fun b -> b = akkaDefaultBranch)) then
+            log "default branch information not available... fetching"
+            directRunGitCommandAndFail srcDir (sprintf "fetch origin %s:%s" akkaDefaultBranch akkaDefaultBranch)
```

Comment:
Makes sense, fetching the `v1.3` branch if that history isn't available on the local machine...